### PR TITLE
View full prepaid card address functionality

### PR DESCRIPTION
**Pull request by DominicSherman**
Screen Shot 2021-08-04 at 14 13 19
Screen Shot 2021-08-04 at 14 13 26

### CODE REVIEW

File: cardstack/src/components/PrepaidCard/PrepaidCard.tsx [hunk 1]
```diff
@@ -340,6 +340,16 @@ const PatternUri = ({
...
 const Top = ({ address, networkName, cardCustomization }: PrepaidCardProps) => {
+  const { navigate } = useNavigation();
+
+  const onPress = () => {
+    navigate(Routes.MODAL_SCREEN, {
+      address,
+      disableCopying: true,
+      type: 'copy_address',
+    });
+  };
```
Comment (danibonilha) [inline cardstack/src/components/PrepaidCard/PrepaidCard.tsx]: we could wrap in an useCallback

File: cardstack/src/components/PrepaidCard/PrepaidCard.tsx [hunk 2]
```diff
@@ -378,13 +388,23 @@ const Top = ({ address, networkName, cardCustomization }: PrepaidCardProps) => {
...
-            variant="shadowRoboto"
-            color={cardCustomization?.textColor as ColorTypes}
-            shadowColor={cardCustomization?.patternColor}
+          <Touchable
+            hitSlop={{
+              top: 2,
+              bottom: 2,
+              left: 2,
+              right: 2,
+            }}
```
Comment (danibonilha) [inline cardstack/src/components/PrepaidCard/PrepaidCard.tsx:397]: Shouldn't this hitslop be a bit bigger ?

**Review by danibonilha — APPROVED** [review]
overall lgtm, could you elaborate why this is conditional ? to enable and disable copying ?

File: cardstack/src/components/PrepaidCard/PrepaidCard.tsx (hunk 2 above)
Comment (DominicSherman) [reply cardstack/src/components/PrepaidCard/PrepaidCard.tsx:397]: Sure, I can make it a bit bigger

**Pull request merged**